doc/whats_new.rst

```diff
@@ -1,6 +1,112 @@
 0.5.0
 =====
 
+    **Released November 2018**
+
+NEW
+---
+
+  :ref:`interactive plotting functions <interactive-plotting>`,
+  eg for use in a notebook.
+
+- New functions :func:`nilearn.plotting.view_surf` and
+  :func:`nilearn.plotting.view_surf` and
+  :func:`nilearn.plotting.view_img_on_surf` for interactive visualization of
+  maps on the cortical surface in a web browser.
+
+- New functions :func:`nilearn.plotting.view_connectome` and
+  :func:`nilearn.plotting.view_markers` to visualize connectomes and
+  seed locations in 3D
+
+- New function :func:`nilearn.plotting.view_img` for interactive
+  visualization of volumes with 3 orthogonal cuts.
+
+:Note: :func:`nilearn.plotting.view_img` was `nilearn.plotting.view_stat_map` in preview builds.
```

Comment:
I did understand what you mean by preview here ?

Reply:
Alpha and Beta builds

Reply:
Are they builds or releases ?

Reply:
Releases. Will clarify.